backend/src/agents/tools/browser_tools.py

```
@@ -0,0 +1,516 @@
+class TextDeleteSchema(BaseModel):
+    text: str = Field(..., description="The text to delete from the input field or textarea.")
+
+class TextDeleteTool(BaseTool):
+    name: str
+    description: str
+    args_schema: type[BaseModel] = TextDeleteSchema
+    page: Page
+
+    async def _run(self, text: str) -> str:
+        try:
+            element = await self.page.query_selector('input[type="text"], textarea')
```

Ruff (0.12.2)

Local variable `element` is assigned to but never used

Remove assignment to unused variable `element`

(F841)